pyproject.toml

```diff
@@ -25,7 +25,6 @@ altair = "^4.0.0"
 beautifulsoup4 = "^4.0.0"
 bleach = "> 3.1.1"
 bookserver = "^1.0.0"
-celery = { extras = ["redis,gevent"], version = "^5.0.0" }
```

Comment:
Shouldn't there be a poetry.lock file corresponding to this change to pyproject.toml ??

Reply:
I added this -- thanks.